geometry_msgs/Pose pose
string block_id
---
Path path